lib/meson.build:34:2: ERROR: Dependency lookup for gtksourceview-5 with method 'pkgconfig' failed: Invalid version, need 'gtksourceview-5' ['>= 5.6'] found '5.4.2'.

--- lib/meson.build.orig	2024-03-04 00:39:28 UTC
+++ lib/meson.build
@@ -31,7 +31,7 @@ libspelling_deps = [
 libspelling_deps = [
   dependency('gio-2.0'),
   dependency('gtk4', version: '>= 4.8'),
-  dependency('gtksourceview-5', version: '>= 5.6'),
+  dependency('gtksourceview-5', version: '>= 5.4'),
 ]
 
 if get_option('enchant').enabled()
